src/type_tracer/analyzer.rs

```diff
@@ -94,6 +95,20 @@ impl RootSymbol {
       &|specifier| self.get_module_from_specifier(specifier),
     )
   }
+
+  pub fn resolve_symbol_dep<'a>(
+    &'a self,
+    module_graph: &ModuleGraph,
+    module: ModuleSymbolRef<'a>,
+    dep: &'a SymbolDep,
+  ) -> Vec<ResolvedSymbolDepEntry<'a>> {
+    super::cross_module::resolve_symbol_dep(
+      module_graph,
+      module,
+      dep,
+      &|specifier| self.get_module_from_specifier(specifier),
+    )
+  }
```

Comment:
Tests for this will be in a follow-up PR: https://github.com/denoland/deno_graph/issues/315